sonar-core/src/main/java/org/sonar/core/i18n/I18nClassloader.java

```diff
@@ -24,6 +24,7 @@
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.util.List;
+import org.sonar.api.Plugin;
 import org.sonar.api.SonarPlugin;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.api.SonarPlugin'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)